.github/workflows/inkomoko.yaml

```diff
@@ -0,0 +1,44 @@
+
+name: Push to ECR Inkomoko
+
+on:
+  push:
+    branches:
```

Comment:
This workflow needs to be kept out of the africa branch and placed directly in inkomoko branch